Common/src/config_structure.cpp

```diff
@@ -2333,6 +2332,23 @@ void CConfig::SetPostprocessing(unsigned short val_software, unsigned short val_
   }
 #endif
 
+  /*--- Set the boolean Wall_Functions equal to true if there is a
+   definition for the wall founctions ---*/
+
+  Wall_Functions = false;
+  if (nMarker_WallFunctions > 0) {
+    for (iMarker = 0; iMarker < nMarker_WallFunctions; iMarker++) {
+      if (Kind_WallFunctions[iMarker] != NO_WALL_FUNCTION)
+        Wall_Functions = true;
+      
+      if ((Kind_WallFunctions[iMarker] == STANDARD_WALL_FUNCTION) || (Kind_WallFunctions[iMarker] == SCALABLE_WALL_FUNCTION)
+        || (Kind_WallFunctions[iMarker] == NONEQUILIBRIUM_WALL_MODEL))
+
+        SU2_MPI::Error(string("For RANS problems, use NO_WALL_FUNCTION, ADAPTIVE_WALL_FUNCTION or EQUILIBRIUM_WALL_MODEL.\n"), CURRENT_FUNCTION);
```

Comment:
How are you defining the current WF approach in terms of enumerated type? I would think it is a STANDARD_WALL_FUNCTION, and if we allow automatic switching on/off based on y^+, I think it would become a SCALABLE_WALL_FUNCTION. At least, I think this is how it is defined in some other software...

Reply:
So, what is ADAPTIVE_WALL_FUNCTION?